x-pack/plugins/security_solution/public/detections/mitre/mitre_tactics_techniques.ts

```diff
@@ -7251,27 +7251,123 @@ export const subtechniques: MitreSubTechnique[] = [
 ];
 
 /**
- * A full object of Mitre Attack Threat data that is taken directly from the `mitre_tactics_techniques.ts` file
+ * An array of full Mitre Attack Threat objects that are taken directly from the `mitre_tactics_techniques.ts` file
  *
  * Is built alongside and sampled from the data in the file so to always be valid with the most up to date MITRE ATT&CK data
  */
-export const getMockThreatData = () => ({
-  tactic: {
-    name: 'Credential Access',
-    id: 'TA0006',
-    reference: 'https://attack.mitre.org/tactics/TA0006',
-  },
-  technique: {
-    name: 'OS Credential Dumping',
-    id: 'T1003',
-    reference: 'https://attack.mitre.org/techniques/T1003',
-    tactics: ['credential-access'],
+export const getMockThreatData = () => [
```

Comment:
Not sure I understand this change to make this function return an array now: I see that the only usage of this function is in `valid_threat_mock.ts`, where it's called as: `getMockThreatData()[0]`, to get the first element.

So what was the purpose of adding the other objects? Is something missing maybe?

Reply:
Sorry - the reference finder in VSCode wasn't working 😓 

Disregard